src/NServiceBus.AcceptanceTests/Recoverability/When_message_is_moved_to_error_queue.cs

```diff
@@ -54,6 +54,30 @@ public Task May_send_outgoing_messages(TransportTransactionMode transactionMode)
             .Run();
         }
 
+        [TestCase(TransportTransactionMode.SendsAtomicWithReceive)]
+        [TestCase(TransportTransactionMode.TransactionScope)]
+        [TestCase(TransportTransactionMode.ReceiveOnly)]
+        [TestCase(TransportTransactionMode.None)]
```

Comment:
The tests needs to have a transaction mode defined because of the setup at Line 100. If we would use a dedicated endpoint for the test, it would use the transport default settings which would be enough for that test. Probably still the best approach since this is not tied to Dtc transports?